### DISCUSSION TITLE
타입스크립트 벡터 관련 질문입니다!

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#305] [5. Q&A]
![Screenshot_2](https://user-images.githubusercontent.com/68143858/152098006-18b9594e-1e1c-4319-b4ef-138663eb8f60.png)

Vector3d 타입들끼리 계산했는데 왜 에러가 나는지 잘 모르겠습니다..

**Comment by mayko1234** [comment]
아래 질문과 같은 이슈가 아닐까 합니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/295

**Reply by hipsc159** [reply]
그러네요 감사합니다!!
